### DISCUSSION TITLE
Support Right to Left chat for Arabic Language

### DISCUSSION THREAD

**Discussion by m7md3b** [Chatterino/chatterino2#3632] [Ideas] [answered]
In Arabic, writing starts from right to left so I hope If there is a way to force Arabic to look normal.

**Comment by LosFarmosCTL** [comment]
See #720

**Comment by Felanbird** [comment, marked as answer]
@m7md3b This was implemented in https://github.com/Chatterino/chatterino2/pull/3958 - I tried to @ everyone who ever mentioned wanting this feature, but it seems I managed to miss you.

**Reply by m7md3b** [reply]
I'm too late, but thank you very much.
